substrate/pkg/controller/substrate/cluster/addons/fluentbitcw.go

```diff
@@ -0,0 +1,71 @@
+/*
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package addons
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/awslabs/kubernetes-iteration-toolkit/substrate/pkg/apis/v1alpha1"
+	"github.com/awslabs/kubernetes-iteration-toolkit/substrate/pkg/utils/kubectl"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+)
+
+var fluentbitcm = `
+apiVersion: v1
+data:
+  cluster.name:  %[1]s
+  http.port: ""
+  http.server: "Off"
+  logs.region: us-west-2
```

Comment:
Can we send logs to the same region as the guest cluster?

Reply:
Today KIT only supports `us-west-2` region, which is why it was defaulted to `us-west-2` earlier.